system/assets/icons/d10-blank.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="40 40 100 100"><path d="m91.39 43.02-30.94 64.95 28.45 18.56 30.78-18.6-28.3-64.9zm1.45-1.07 28.6 65.31 16.56-5.35 1.07-26.4-46.23-33.56zm-2.96 86.09v10l47.6-33.59-15.9 4.47-31.7 19.12zm0-85.8-31.25 65.09-16.27-7-1.43-26.49 48.95-31.6zm-2.16 85.59v10.22l-45.57-35.22 15.79 6.16 29.78 18.84z" /></svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="100" height="100" viewBox="40 40 100 100"><path d="m91.39 43.02-30.94 64.95 28.45 18.56 30.78-18.6-28.3-64.9zm1.45-1.07 28.6 65.31 16.56-5.35 1.07-26.4-46.23-33.56zm-2.96 86.09v10l47.6-33.59-15.9 4.47-31.7 19.12zm0-85.8-31.25 65.09-16.27-7-1.43-26.49 48.95-31.6zm-2.16 85.59v10.22l-45.57-35.22 15.79 6.16 29.78 18.84z" /></svg>
```

Comment:
`viewBox="40 40 100 100"` has an implied height of 60 units. doesn't really matter on this one since the aspect ratio is the same anyways, but the result here (relative to the source) is different, so i thought i'd point that out in case anyone reading this ends up adjusting some SVGs in the future ;)

Reply:
oh, dang, you're right. 

> But I already feel dirty for adding width and height to _**scalable vector graphics**_.

my understanding is that they're not like `height`/`width` attributes on e.g. the HTML `<img>` element (which usually requires units). instead, they operate on the same coordinate system as everything else in SVG. so they ought to scale just fine.

https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/height